Create License file

jhthompson opened the pull request:
I think this is the license we wanted to use?

Commit:
Updated landing page to have new content
Menu button now fixed on the side of the page
Map doesn't take up full screen

Commit:
Fixed issue with overlay not displaying
Added "cards" to the homepage

Verkir committed:
Minor CSS fixes

Verkir committed:
Changed size of time picker on certain devices

Commit:
Added help button and page
Added navbar

Verkir committed:
Added prototype map

Verkir committed:
Merge branch 'gh-pages' of https://github.com/upeiSNACKS/snacks into gh-pages

rjarsenault1101 committed:
Added AJAX, modified code to be more general

Verkir committed:
Merge pull request #20 from upeiSNACKS/gh-pages

Merging

Verkir committed:
Sample  Map

rjarsenault1101 committed:
Added automatic GEOJSON usage

rjarsenault1101 committed:
Merge branch 'APIPHP' into gh-pages

Verkir committed:
Merge branch 'voronoi' into gh-pages

rjarsenault1101 committed:
removed merge stuff

jhthompson committed:
Move sidebar js to new file for pages other than index

jhthompson committed:
Enabling sidebar functionality in pages other than index

Verkir committed:
Merge pull request #22 from upeiSNACKS/sidebar-alt-pages

Sidebar button wasn't working on pages other than index

jhthompson committed:
Update about.html

jhthompson committed:
Writing more about the general project

jhthompson committed:
Add page describing prototype device

jhthompson committed:
Adding prototype device wiring and updating prototype device page

rjarsenault1101 committed:
added dynamic popups, removed useless files

rjarsenault1101 committed:
Merge branch 'gh-pages' into APIPHP

rjarsenault1101 committed:
Merge branch 'gh-pages' into APIPHP

Verkir committed:
Merge pull request #24 from upeiSNACKS/APIPHP

added dynamic popups, removed useless files

rjarsenault1101 committed:
Merge branch 'gh-pages' of https://github.com/upeiSNACKS/snacks into gh-pages

rjarsenault1101 committed:
Merge branch 'gh-pages' of https://github.com/upeiSNACKS/snacks into gh-pages

rjarsenault1101 committed:
Added a signup form, along with client side validation and CSS as well

rjarsenault1101 committed:
added more functionality

jhthompson committed:
Tidying about page

jhthompson committed:
Create License file

jhthompson committed:
Continuing work on about page

Verkir commented:
I think that license is fine. We need to add the block into each source file in addition to having this file as stated near the bottom. For author I'm not sure if we put them all under one name/team name or the author of that specific file.

rjarsenault1101 committed:
more changes

jhthompson committed:
Update License with applicable group information

rjarsenault1101 committed:
more changes

rjarsenault1101 committed:
added stuff again

rjarsenault1101 committed:
finished form validation

rjarsenault1101 committed:
additional features added

mregger committed:
feature(api docs) added interactive API documentation, which includes a custom url builder

mregger committed:
Merge pull request #29 from upeiSNACKS/interactive-api-docs

feature(api docs) added interactive API documentation, which includes…

Commit:
Start of cards fixes

Commit:
Fix merge conflicts
Improvements to api.html
New version of fontaweseome and fixed broken icons

rjarsenault1101 committed:
phpmailer functionality

rjarsenault1101 committed:
added email to send

rjarsenault1101 committed:
Merge branch 'gh-pages' into application-form

rjarsenault1101 committed:
forgot to save lol

Verkir committed:
Cards now use data from the API

rjarsenault1101 committed:
added stuff

Verkir committed:
Setup favicon and title for every page

rjarsenault1101 committed:
finished email

rjarsenault1101 committed:
email works.

Verkir committed:
CSS Updates

Will probably need to discuss certain changes and do some more testing to see what is best.

rjarsenault1101 committed:
email complete. Fixed some js

rjarsenault1101 committed:
Merge branch 'gh-pages' into application-form

Verkir committed:
Added a page with an alternate form

jhthompson committed:
Finishing writing on the About page

jhthompson committed:
Add battery discussion into the protype page

jhthompson committed:
Update prototype.html

jhthompson committed:
Merge branch 'gh-pages' into about-page

jhthompson committed:
Merge pull request #32 from upeiSNACKS/about-page

Add about page and prototype device page - will create new branch to modify styles and update the old pages with links to the prototype page

jhthompson committed:
Update prototype navbar style to match other pages

copied and renamed api.html and replaced the content div with prototype content

jhthompson committed:
Add link to prototype page on all pages

jhthompson committed:
Merge pull request #33 from upeiSNACKS/style-and-link-fixes

Style and link fixes

Verkir committed:
Added Help modal to everypage and removed unused files

rjarsenault1101 committed:
Merge branch 'gh-pages' into application-form

Verkir committed:
Text fixes

rjarsenault1101 committed:
Merge branch 'gh-pages' into application-form

Commit:
Continued work from the Chroropleth example

Commit:
Removed setters and getters for global variables

jhthompson committed:
Spelling fix

Checout -> Check out

rjarsenault1101 committed:
fixed signup sheet. added some CSS

rjarsenault1101 committed:
fixed lat/lon validation and links

rjarsenault1101 committed:
fixed lat/lon issue

rjarsenault1101 committed:
Merge branch 'gh-pages' into application-form

Verkir committed:
Fixed typo and spacing

Verkir committed:
Merge pull request #34 from upeiSNACKS/application-form

Application form

Commit:
Fixed the color and legend

Commit:
Merge branch 'voronoi' of https://github.com/upeiSNACKS/snacks into voronoi

Verkir committed:
Merge branch 'voronoi' into gh-pages

Commit:
Removed call of non-existing JS function

Commit:
Fixed overlay problem

rjarsenault1101 committed:
fixed small issues

Commit:
Changed legend size

jhthompson committed:
Modify LICENSE with only relevant info

jhthompson committed:
Merge pull request #38 from upeiSNACKS/gh-pages

Getting updated code before adding license info

jhthompson committed:
Add license information to all files

I may have missed some files - especially in the documentation folder, but I think most of that stuff isn't source code and doesn't require the license header at the start

Verkir reviewed (approved):
Looks good to me
